velox/dwio/common/tests/utils/BatchMaker.cpp

```diff
@@ -188,7 +188,7 @@ VectorPtr BatchMaker::createVector<TypeKind::HUGEINT>(
       size,
       gen,
       [&gen]() {
-        return HugeInt::build(Random::rand32(gen), Random::rand32(gen));
+        return HugeInt::build(Random::rand64(gen), Random::rand64(gen));
```

Comment:
Add the commit of https://github.com/facebookincubator/velox/pull/6342 into this PR as a separate commit, and put it before adding the benchmarks